qubes-rpc/qvm-copy

```diff
@@ -93,8 +93,12 @@ else
   VM="@default"
 fi
 
+# strip trailing slash from source directory
+SOURCE="${@%/}"
```

Comment:
This breaks handling multiple paths. But you can keep it being an array with `SOURCE=( "${@%/}" )`, and later use "${SOURCE[@]}".